rmgpy/reaction.py

```diff
@@ -308,99 +309,101 @@ def to_cantera(self, species_list=None, use_chemkin_identifier=False):
+            ct_reaction = ct.Reaction(reactants=ct_reactants, products=ct_products, rate=ct.ChebyshevRate())
 
+        elif isinstance(self.kinetics, ThirdBody):
+            if ct_collider is not None:
+                ct_reaction = ct.ThreeBodyReaction(reactants=ct_reactants, products=ct_products, third_body=ct_collider)
             else:
-                raise NotImplementedError('Unable to set cantera kinetics for {0}'.format(self.kinetics))
-
-            # Set reversibility, duplicate, and ID attributes
-            if isinstance(ct_reaction, list):
-                for rxn in ct_reaction:
-                    rxn.reversible = self.reversible
-                    # Set the duplicate flag to true since this reaction comes from multiarrhenius or multipdeparrhenius
-                    rxn.duplicate = True
-                    # Set the ID flag to the original rmg index
-                    rxn.ID = str(self.index)
+                ct_reaction = ct.ThreeBodyReaction(reactants=ct_reactants, products=ct_products)
+
+        elif isinstance(self.kinetics, Troe):
+            if ct_collider is not None:
+                ct_reaction = ct.FalloffReaction(
+                    reactants=ct_reactants,
+                    products=ct_products,
+                    tbody=ct_collider,
+                    rate=ct.TroeRate()
+                )
             else:
-                ct_reaction.reversible = self.reversible
-                ct_reaction.duplicate = self.duplicate
-                ct_reaction.ID = str(self.index)
+                ct_reaction = ct.FalloffReaction(
+                    reactants=ct_reactants,
+                    products=ct_products,
+                    rate=ct.TroeRate()
+                )
+
+        elif isinstance(self.kinetics, Lindemann):
+            if ct_collider is not None:
+                ct_reaction = ct.FalloffReaction(
+                    reactants=ct_reactants,
+                    products=ct_products,
+                    tbody=ct_collider,
+                    rate=ct.LindemannRate()
+                )
+            else:
+                ct_reaction = ct.FalloffReaction(
+                    reactants=ct_reactants,
+                    products=ct_products,
+                    rate=ct.LindemannRate()
+                )
 
-            self.kinetics.set_cantera_kinetics(ct_reaction, species_list)
+        elif isinstance(self.kinetics, SurfaceArrhenius):
+            ct_reaction = ct.InterfaceReaction(reactants=ct_reactants,
+                                                products=ct_products,
+                                                rate=ct.InterfaceArrheniusRate())
 
-            return ct_reaction
+        elif isinstance(self.kinetics, StickingCoefficient):
+            ct_reaction = ct.Reaction(reactants=ct_reactants,
+                                        products=ct_products,
+                                        rate=ct.StickingArrheniusRate())
```

Comment:
Optional fix but I prefer the indentation styles used for Lindemann and Troe above and think SurfaceArrhenius and StickingCoefficient should match those.